test/integration/update_template_content_test.go

```diff
@@ -96,20 +99,23 @@ func (s *UpdateTemplateContentSuite) TestCreateCandlepinContent() {
 
 	// Create initial template
 	reqTemplate := api.TemplateRequest{
-		Name:            pointy.Pointer("test template"),
+		Name:            pointy.Pointer(fmt.Sprintf("test template %v", rand.Int())),
```

Comment:
I randomized the name here, otherwise i couldn't actually re-run the test without clearing the db